### PR TITLE
delay docstring update

### PR DESCRIPTION
**Pull request by davewiththenicehat**
delay docstring update

The old one had not been properly updated or formatted.

#### Brief overview of PR changes/additions
An update to utils.delay's docstring

#### Motivation for adding to Evennia
I had it formatted incorrectly. Also it's return message was incorrect.

### CODE REVIEW

File: evennia/utils/utils.py [hunk 1]
```diff
@@ -1021,44 +1021,34 @@ def uses_database(name="sqlite3"):
...
+    be called for persistent or non-persistent tasks.
+
+    If persistent is set to True.
+        The callback, its arguments and other keyword arguments will be saved
+        (serialized) in the database, assuming they can be. The callback will
+        be executed even after a server restart/reload, taking into account the
+        specified delay (and server down time).
+        Keep in mind that persistent tasks arguments and callback should not
+        use memory references.
+        All task's whose time delays have passed will be called on server startup.
```
Comment (Griatch) [inline evennia/utils/utils.py:1036]: This sounds like it belongs partly in the `persistent` arg description and in a `Notes` section.

File: evennia/utils/utils.py [hunk 2]
```diff
@@ -1021,44 +1021,34 @@ def uses_database(name="sqlite3"):
...
             after `timedelay` seconds.
         args (any, optional): Will be used as arguments to callback
+
     Keyword Args:
-        persistent (bool, optional): Should make the delay persistent
-            over a reboot or reload. Defaults to False.
-        any (any): Will be used as keyword arguments to callback.
+        persistent (bool, optional): should make the delay persistent
+            over a reboot or reload
+        any (any, optional): Will be used as keyword arguments to callback.
```
Comment (Griatch) [inline evennia/utils/utils.py:1047]: It's actually OK to just put `any:` I think.

**Review by Griatch — CHANGES_REQUESTED** [review]
Not sure why you moved this text up from the `Note` to the top of the function?

**Comment by davewiththenicehat** [comment]
What tool is being used to generate the documentation?
I think I got the kwargs section incorrect also.

I'm happy to read up on it. I don't know where to look this up. The google style docstring documentation I was able to find did not show 'Notes' as a header type so I removed it.

**Comment by Griatch** [comment]
We use sphinx with the Napoleon parser. Easiest is probably to look at existing Evennia docstrings, but the Napoleon parser is documented here: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

**Comment by Griatch** [comment]
As far as I can tell the existing doc string was fine, so closing this.

**Comment by davewiththenicehat** [comment]
delay does not return a deferred or int. It returns a TaskHandlerTask instance.

**Pull request merged**